### ISSUE TITLE
Decide whether or how to standardize posInSet and setSize calculation where a subset of items are in one or more fieldsets

### ISSUE THREAD

**Issue by mcking65** [w3c/aria#2791]
#### Problem

When some but not all of the items in a menu are part of a group or fieldset, it is not clear how aria-setsize and aria-posinset should be set. For example, when some of the items in a menu are part of a set of `menuitemradio` elements, there are multiple, valid ways of setting aria-setsize and aria-posinset, each with a different set of pros and cons. This ambiguity negatively impacts interoperability and AT user experiences.

For brevity, a fieldset will be refered to as a group since fieldsets are mapped to group in ARIA.

#### Why it Matters Now

When browser start supporting [native application menus as proposed in OpenUI](https://open-ui.org/components/menu.explainer/), they will need a default way of calculating  values for [aria-posinset](https://w3c.github.io/aria/#aria-posinset) and [aria-setsize](https://w3c.github.io/aria/#aria-setsize). A thoughtful approach to determining the default algorithm will be beneficial to AT users and facilitate interoperability.

This issue is inspired by conversation in a [May 14, 2026 ARIA Deep Dive meeting on the OpenUI proposal](https://www.w3.org/events/meetings/9283d30f-dcca-4303-b207-fe8455df56c4/) where [ARIA issue 2782](https://github.com/w3c/aria/issues/2782) was discussed.

#### Common Approaches

1. Menu-Based Indexing:
    * aria-setsize is the total number of items in the menu container. 
    * aria-posinset is the position of the item in the menu container.
    * Groups within the menu do not influence values of aria-setsize or aria-posinset.
2. Group-Adjusted Indexing:
    * If an item is contained in a group, aria-setsize is the number of items in the group and aria-posinset is the position within that group.
    * If an item is not contained in a group:
        * aria-setsize is the number of consecutive items in the subset of items between the nearest menu and group boundaries, e.g., the start of the menu and the start of the first group, the end of a group and the start of the next group, or the end of a group and the end of the menu.
        * aria-posinset is the position of the item within the subset defined by the subset boundaries.

#### Example

An edit menu contains 11 items. Eight of the items are contained within three different groups. The remaining 3 items are positioned separately, outside of any group.

Item Number|Item Name|Group Name|Menu-Based Position/Set Size|Group-Adjusted Position/Set Size
---|---|---|---|---
1|Insert Link|N/A|1 / 11|1 / 1
2|Bold|Style|2 /  11|1 / 3
3|Underline|Style|3 / 11|2 / 3
4|Strike Through|Style|4 / 11|3 / 3
5|Heading|N/A|5 / 11|1 / 1
6|Decrease Size|Font Size|6 / 11|1 / 2
7|Increase Size|Font Size|7 / 11|2 / 2
8|Left|Alignment|8 / 11|1 / 3
9|Center|Alignment|9 / 11|2 / 3
10|Right|Alignment|10 / 11|3 / 3
11|Clear Formatting|N/A|11 / 11|1 / 1

#### Objective

Form consensus around a specific method of calculating position and set size values and update the relevant ARIA specifications to align with that consensus. The method could be:
1. Menu-based indixing.
2. Group-adjusted indexing.
3. Another approach TBD that may or may not include additional new, not-yet-defined properties.

**Comment by mcking65** [comment]
To effectively discuss thistopic, we need representation from multiple screen reader developers and users. Recommend, at a minimum, we include:
* James Teh
* Brett Lewis
* James Craig
* At least 3 full time screen reader users
